jekyll/_cci2/workflows.adoc

```diff
@@ -423,15 +487,17 @@ workflows:
               only: /^v.*/
 ----
 
-In the example below, two jobs are defined within the `build-n-deploy` workflow:
+---
+
+In the example below, two jobs are configured within the `build-deploy` workflow:
 
 * The `build` job runs for all branches and all tags.
 * The `deploy` job runs for no branches and only for tags starting with 'v'.
```

Comment:
This is incorrect, and in contradiction with the description under the "Using filters in your workflows" section
```
If both branch and tag filtering is configured and a push to your code includes both branch and tag information, the branch filters take precedence.
```

In the example, both branch and tag filtering is configured so only the branch filtering would apply.

Reply:
Oh yeah! Ok, it sounds like the example this is saying it shows is not actually possible to configure? I will simplify and remove the branch filter as you suggest. Thanks for spotting this!